src/ParsePushAdapter.js

```diff
@@ -49,8 +51,19 @@ export class ParsePushAdapter {
     let deviceMap = classifyInstallations(installations, this.validPushTypes);
     let sendPromises = [];
     for (let pushType in deviceMap) {
-      let sender = this.senderMap[pushType];
+      var sender = this.senderMap[pushType];
       let devices = deviceMap[pushType];
+      
+      if (!sender) {
+        if (pushType == 'tvos') {
```

Comment:
I added this as it was suggested in the related issue. However I also think it is better to not add redundancy and require the admin to define a certificate for each plattform he wants to support.